fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/derby/DerbyMaster.java

```diff
@@ -56,22 +51,7 @@
     public DerbyMaster(String database) {
         this.database = database;
 
-        try (PrintWriter out = new PrintWriter(DERBY_PROPERTIES)){
-            Class.forName(DERBY_TRANSLATOR.getDriverClassName());
```

Comment:
I take it the translator already does the class name lookup?  otherwise if it doesn't I think this should be done on line 53. it's the general jdbc impl class loading pattern

Reply:
yeah, it's interesting that removing this has been working well. checking if this is really needed here ....

Reply:
did some search, JDBC 4.0 (and after) drivers that are found in your class path are automatically loaded, only JDBC drivers older than 4.0 have to be loaded by Class.forName.
I will keep it just in case JDBC older drivers are/(will be) still used...

Reply:
added it back with comment even thought I believe it's not really needed now

Reply:
it's best to maintain a consistent pattern - it's in FhirDbDaoImpl too